content/en/security_monitoring/detection_rules/security_monitoring.md

```diff
@@ -0,0 +1,165 @@
+
+Add additional queries with the Add Query button.
+
+**Note**: The query applies to all Datadog events and ingested logs which do not require indexing.
+
+#### Joining queries
+
+Joining together logs that span a timeframe can increase the confidence or severity of the Security Signal. For example, to detect a successful brute force attack, both successful and unsuccessful authentication logs must be correlated for a user.
+
+{{< img src="security_monitoring/detection_rules/joining_queries_define.png" alt="Define search queries"  >}}
+
+The Detection Rules join the logs together using a group by value. The group by values are typically entities (e.g. IP address, user, etc), but can be any attribute.
+
+{{< img src="security_monitoring/detection_rules/group_by.png" alt="Group by"  >}}
+
+The Detection Rule cases join these queries together based on their group by value. The group by attribute is typically the same attribute because the value must be the same for the case to be met. If a group by value doesn’t exist, the case will never be met. A Security Signal is generated for each unique group by value when a case is matched.
+
+{{< img src="security_monitoring/detection_rules/set_rule_cases2.png" alt="Set rule cases"  >}}
+
+In this example, when greater than 5 failed logins and a successful login exist for the same `@usr.name`, the first case is matched and a Security Signal is generated.
+
+{{< img src="security_monitoring/detection_rules/gbv2.png" alt="Set rule cases" >}}
+
+[1]: /logs/search_syntax/
+{{% /tab %}}
+
+{{% tab "New Term" %}}
+
+### Search query
+
+{{< img src="security_monitoring/detection_rules/new_term.png" alt="Define the search query" >}}
+
+Construct a search query using the same logic as a [log explorer search][1]. Each query has a label, which is a lowercase ASCII letter. The query name can be changed from an ASCII letter by clicking the pencil icon.
+
+**Note**: The query applies to all Datadog events and ingested logs which do not require indexing.
+
+#### Learned value
+
+{{< img src="security_monitoring/detection_rules/learning_duration.png" alt="Define the learned value" >}}
+
+Select a value to detect, learning duration, and, optionally, define a signal grouping. The defined group-by generates a signal for each group by value. Typically, the group by is an entity (e.g. user, IP, etc.).
+
+For example, create a query for successful user authentication and set **detect new value** to `country` and group by to `user`. Set a learning duration of `7 days`. Once configured, logs coming in over the next 7 days are evaluated with the set values. If a log comes in with a new value after the learning duration, a signal is generated and the new value is learned to prevent future signals with this value.
```

Comment:
```suggestion
For example, create a query for successful user authentication and set **detect new value** to `country` and group by to `user`. Set a learning duration of `7 days`. Once configured, logs coming in over the next 7 days are evaluated with the set values. If a log comes in with a new value after the learning duration, a signal is generated, and the new value is learned to prevent future signals with this value.
```